spec/models/user/api_spec.rb

```diff
@@ -55,6 +55,14 @@
         expect(api_user.valid_for).to eq(v)
       end
     end
+
+    context 'personal_team' do
+
+      it 'does not create personal_team for API user' do
+        user = User::Api.create!(username: 'APIAlice', human_user: users(:alice).dup)
```

Comment:
create api user by method provided on User::Human instead of own implementation here.
a user can have multiple api users ... so assigning it to alice should be fine